src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/v2/common.kt

```diff
@@ -288,6 +311,20 @@ fun handleXpayRefundResponse(
   }
 }
 
+fun handleNpgRefundResponse(
+  transaction: BaseTransactionWithRequestedAuthorization,
+  refundResponse: RefundResponseDto,
+  transactionsEventStoreRepository: TransactionsEventStoreRepository<TransactionRefundedData>,
+  transactionsViewRepository: TransactionsViewRepository
+): Mono<BaseTransaction> {
+  logger.info(
+    "Refund for transaction with id: [{}] and NPG operationId [{}] processed successfully",
+    transaction.transactionId.value(),
+    refundResponse.operationId!!)
```

Comment:
Why this not-null assertion on the received response operationId field? 
If not received this line will throw NullPointerException instead of logging null as operationId field.
Maybe null received operationId can be handle raising a proper error (if it is the case) but I would remove this check performed here since it's just a log line.
What do you think?

Reply:
If this value is null, maybe something went wrong in NPG. Of course, we can raise an exception here.